lua/telescope/_extensions/file_browser/finders.lua

```diff
@@ -149,6 +149,8 @@ fb_finders.finder = function(opts)
     close = function(self)
       self._finder = nil
     end,
+    custom_prompt_title = opts.custom_prompt_title,
```

Comment:
For simplicity, I think we can just call this `{prompt, results}_title`? They'll only ever be set for custom titles.

Reply:
Agreed, I simplified the names as you suggested.